lldb-eval/parser.cc

```diff
@@ -2133,6 +2154,36 @@ ExprResult Parser::ParseBooleanLiteral() {
       /*is_literal_zero*/ false);
 }
 
+ExprResult Parser::ParseCharLiteral() {
+  ExpectOneOf(clang::tok::char_constant, clang::tok::wide_char_constant,
+              clang::tok::utf16_char_constant, clang::tok::utf32_char_constant);
+  clang::SourceLocation loc = token_.getLocation();
+
+  llvm::SmallString<8> CharBuffer;
+  bool CharInvalid = false;
+  llvm::StringRef ThisTok = pp_->getSpelling(token_, CharBuffer, &CharInvalid);
+  clang::CharLiteralParser charLiteral(ThisTok.begin(), ThisTok.end(), loc,
```

Comment:
We follow Google C++ style guide and use `camel_case` for local variables -- https://google.github.io/styleguide/cppguide.html#Variable_Names